packages/client/components/AddTeamMemberModal.tsx

```diff
@@ -48,13 +87,35 @@ const ErrorMessage = styled(StyledError)({
   marginTop: '.5rem'
 })
 
-const ShareableLink = styled('div')({
+const StyledLabelHeading = styled(LabelHeading)({
+  alignItems: 'center',
   display: 'flex',
-  justifyContent: 'center',
-  // padding (18+8) addresses hanging icon at mobile widths
-  padding: '0 26px'
+  fontSize: 15,
+  lineHeight: '21px',
+  padding: '0 0 16px',
+  textTransform: 'none'
 })
 
+const StyledCopyShortLink = styled(CopyShortLink)({
+  borderRadius: 4,
+  border: `1px dashed ${PALETTE.EMPHASIS_COOL_LIGHTER}`,
+  color: PALETTE.EMPHASIS_COOL,
+  fontSize: 15,
+  margin: '0 0 32px',
+  padding: 11,
+  ':hover': {
+    color: PALETTE.EMPHASIS_COOL_LIGHTER
+  }
+})
+
+const IllustrationBlock = () => {
+  const showIllustration = useBreakpoint(INVITE_DIALOG_WITH_ILLUS)
+  const imageSrc =
+    'https://s3.amazonaws.com/action-files.parabol.co/static/illustrations/illus-momentum.png' ||
```

Comment:
@mattkrick the `__STATIC_IMAGES__` didn’t seem to work locally. What’s funny is it works locally for the credit card images. Also, I noticed in the dev webpack config that it points to `/static/images` and I’m not sure the `/images` directory is a thing on S3.

Reply:
so in dev it'll point to action/static/images, in prod, it'll point to S3/static/. in order to make it work in both, you'll want to uplaod the file to the /static/images folder in the repo & then again in the S3/static folder.